session.go

```diff
@@ -5212,3 +5226,67 @@ func hasErrMsg(d []byte) bool {
 	}
 	return false
 }
+
+// getRFC2253NameString converts from an ASN.1 structured representation of the certificate
+// to a UTF-8 string representation(RDN) and returns it.
+func getRFC2253NameString(certificate *x509.Certificate) string {
+	var RDNElementsASN1 = pkix.RDNSequence{}
+	asn1.Unmarshal(certificate.RawSubject, &RDNElementsASN1)
+
+	var RDNElementsString = []string{}
+	for i := len(RDNElementsASN1) - 1; i >= 0; i-- {
```

Comment:
Is there a reason to iterate from the back? DN ordering?
If so this will need a comment

Reply:
the sequence is in reverse order, in the RFC it says "the output consists of the string encodings of each RelativeDistinguishedName in the RDNSequence, starting with the last element of the sequence and moving backwards toward the first."